pytrading212/trading212.py

```diff
@@ -12,16 +12,16 @@
 from selenium import webdriver
 from selenium.common.exceptions import NoSuchElementException
 from selenium.webdriver.common.by import By
-from selenium.webdriver.support import expected_conditions
+from selenium.webdriver.support import expected_conditions, wait
 from selenium.webdriver.support.ui import WebDriverWait
 
 from pytrading212.order import ValueOrder
 from pytrading212.position import Position
 
 
 class Mode(Enum):
-    DEMO = "demo"
-    LIVE = "live"
+    DEMO = ("demo",)
```

Comment:
Why are you using round brackets?

Reply:
No idea, I think those must have been in the original code, I didn't intentionally add it!

Reply:
Yes, maybe you're using an older version.